lib/swagger-client.js

```diff
@@ -2674,8 +2694,7 @@ SwaggerOperation.prototype.urlify = function (args) {
     param = params[i];
     if(param.paramType === 'query') {
       var addition = '';
-      if (Array.isArray(param)) {
-        var j;   
+      if (Array.isArray(param)) { 
```

Comment:
it was introduced in 209019f1390f1652926a9446398244ccef2f7976, and not part of my change, if you want I can change it